### PR TITLE
Updated error code and MOSIP-23343

### PR DESCRIPTION
**Pull request by ase-101**

### CODE REVIEW

File: MockMDS/src/main/java/io/mosip/mock/sbi/service/SBIServiceResponse.java [hunk 1]
```diff
@@ -155,7 +155,7 @@ public String processDeviceDicoveryInfo (SBIMockService mockService)
         	 DeviceDiscoveryRequestDetail requestObject = (DeviceDiscoveryRequestDetail) getRequestJson (SBIConstant.MOSIP_DISC_VERB);
         	 String type = null;
         	 if (requestObject != null && requestObject.getType() != null && requestObject.getType().length() > 0)
-        		 type = requestObject.getType().toString ().trim().toLowerCase();
+        		 type = requestObject.getType().toString ().trim();
```
Comment (gsasikumar) [inline MockMDS/src/main/java/io/mosip/mock/sbi/service/SBIServiceResponse.java:158]: Is there any issue in converting to lowercase?

**Review by gsasikumar — CHANGES_REQUESTED** [review]
@ase-101 can you chk the comments. Also Can on the error codes is there a change? can you point to the document?

File: MockMDS/src/main/java/io/mosip/mock/sbi/service/SBIServiceResponse.java (hunk 1 above)
Comment (ase-101) [reply MockMDS/src/main/java/io/mosip/mock/sbi/service/SBIServiceResponse.java]: Not in getType, we have reverted this and removed the same toLowerCase() on getDeviceStatus() method. As further comparisons of device status are case-sensitive.

Comment (JanardhanBS-SyncByte) [reply MockMDS/src/main/java/io/mosip/mock/sbi/service/SBIServiceResponse.java]: 109 error Requested number of biometric (Finger/IRIS) not supported already defined in the spec which is required to test in complaince toolkit

**Pull request merged**